pkg/hub/submarinerbrokerinfo/brokerinfo_test.go

```diff
@@ -200,12 +200,13 @@ var _ = Describe("Function Get", func() {
 							SubmarinerRouteAgentImagePullSpec:          "quay.io/submariner/submariner-route-agent:10.0.1",
 							SubmarinerNetworkPluginSyncerImagePullSpec: "quay.io/submariner/submariner-networkplugin-syncer:10.0.1",
 						},
-						CableDriver:         "wireguard",
-						IPSecNATTPort:       5678,
-						NATTEnable:          true,
-						LoadBalancerEnable:  true,
-						AirGappedDeployment: true,
-						GlobalCIDR:          "242.1.0.0/16",
+						CableDriver:              "wireguard",
```

Comment:
While we are at it, please add a check in Line 177 to validate the default value as well.